ldk/javascript/src/whisper/types.ts

```diff
@@ -238,13 +240,35 @@ export type Message = WhisperComponent<WhisperComponentType.Message> & {
 };
 
 export type DropZone = WhisperComponent<WhisperComponentType.DropZone> & {
+  /**
+   * Specify what extensions the user can include. Do not include the dot.
+   *
+   * @example ['jpg','jpeg','png']
+   */
   accept?: string[];
   label: string;
+  /**
+   * The number of files that can be selected. There is a hard limit of ten files that can be selected at once.
+   */
   limit?: number;
+  /**
+   * If provided, replaces the word "files" in the component.
+   */
   noun?: string;
+  /**
+   * The callback function to call whenever the user selects or unselects a file.
+   */
   onDrop: WhisperHandlerWithParam<File[]>;
   tooltip?: string;
   validationError?: string;
+  /**
+   * The value property can be used to update the list of files selected, including dropping them. If the value property
```

Comment:
You can probably just write the first sentence as: _The value property can be used to modify the selected files._ For some reason "including dropping them" is bending my mind (you mean remove right? I haven't had coffee yet, send help). Anyway, the next few sentences clearly state what you mean :)